tools/depends/target/libmicrohttpd/Makefile

```diff
@@ -35,13 +35,13 @@ $(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE) $(DEPS)
 	cd $(PLATFORM); $(CONFIGURE)
 
 $(LIBDYLIB): $(PLATFORM)
-	$(MAKE) -C $(PLATFORM)/src/microhttpd
-	$(MAKE) -C $(PLATFORM)/src/include install
+	$(MAKE) -j1 -C $(PLATFORM)/src/microhttpd
+	$(MAKE) -j1 -C $(PLATFORM)/src/include install
```

Comment:
because it fails when doing it multi threaded - at least it did in 4 of 5 cases ... i can limit it to darwin if you want